htmresearch/frameworks/nlp/classification_model.py

```diff
@@ -181,7 +181,9 @@ def inferDocument(self, document, returnDetailedResults=False,
                                      sortResults=False)
 
       if votes.sum() > 0:
-        categoryVotes[votes.argmax()] += 1
+        # Increment the most likely category, breaking ties in a random fashion
```

Comment:
`sortResults` is for `_inferDocumentDetailed()`. The sorting here is to the same effect as `votes.argmax()` -- to get the maximum category so we can increment its votes. I'll change the naming so it's less deceiving.